corehq/apps/registration/templates/registration/confirmation_complete.html

```diff
@@ -1,34 +0,0 @@
-{% extends "registration/base.html" %}
-
-{% if valid_confirmation %}
-{% block js-inline %}{{ block.super }}
-<script>
-    {% if not is_error %}
-    $(function () {
-        analytics.workflow("Confirmed new project");
```

Comment:
if we wanted we could push this to python, right? not sure how much of a priority is.